src/actions/event-actions.js

```diff
@@ -847,6 +847,7 @@ export const getEventFeedback =
         const {currentSummitState} = getState();
         const accessToken = await getAccessTokenSafely();
         const {currentSummit} = currentSummitState;
+        const summitTZ = currentSummit.time_zone.name;
```

Comment:
@romanetar check this https://github.com/fntechgit/summit-admin/commit/6a3dee53957e7447e1bb55bfdee8975c601512e2
we user time_zone_id
regards

Reply:
done